__tests__/task.test.js

```diff
@@ -56,9 +50,8 @@ describe('Task', () => {
       expect(statusCode).toBe(200);
     });
     it('should return 200 on edit/:id ', async () => {
-      const existingTask = await app.objection.models.task.query().insert({
-        name: 'test',
-        description: 'test',
+      const existingTask = await db.insert.task({
+        ...create.task(),
         statusId: existingStatus.id,
         creatorId: existingUser.id,
       });
```

Comment:
the same, fields should be build inside builder

Reply:
done